### PR TITLE
Add dark theme

### PR DESCRIPTION
**Pull request by Aman-Codes**
## Proposed changes
Added the following enhancements in E2E dashboard:
- Added dark theme
- Removed status badge from all workflows
- Added rotating spinner for currently running jobs
- Removed build and release pipeline runs from all workflow runs
- Added new status badge

## Screenshots

image

image

TableDark

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus-e2e/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] I have added necessary documentation (if appropriate)
- [x] I have added Screenshots or Terminal snippets in the PR comment to validate the successful execution of changes

**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] data-cy=* parameters have been added in codebase ( ChaosCenter Tests )
- [ ] data-cy=* parameters need to be added in codebase ( ChaosCenter Tests )
- [x] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. uncaught exceptions from 3rd party library (if any)

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

**Comment by uditgaurav** [comment]
Instead of an empty spinner running on the table can we make the spinner of 4 dynamic? similar thing on steps as well.

image

**Comment by Aman-Codes** [comment]
@uditgaurav Updated it to 
image

**Comment by Aman-Codes** [comment]
@uditgaurav Center aligned heading and reduced its font size
image

**Comment by uditgaurav** [comment]
@Aman-Codes it would be good if we reduce the size of what we were having already. cc: @Jonsy13 @oumkale

File: src/containers/layouts/Scaffold/styles.js [hunk 1]
```diff
@@ -14,11 +14,11 @@ const useStyles = makeStyles((theme) => ({
     },
     "& ::-webkit-scrollbar-track": {
       marginTop: theme.spacing(1),
-      webkitBoxShadow: `inset 0 0 8px ${theme.palette.common.black}`,
-      backgroundColor: theme.palette.border.main,
+      webkitBoxShadow: `inset 0 0 8px ${theme?.palette?.common?.black}`,
```
Comment (Jonsy13) [inline src/containers/layouts/Scaffold/styles.js:17]: Why are we using `?` with theme, it should always be there right? Why are we checking if it is undefined?

File: src/components/CustomCard/index.jsx [hunk 2]
```diff
@@ -12,47 +12,53 @@ import { Icon } from "litmus-ui";
...
-import { ReactComponent as SkippedIcon } from "svg/Skipped.svg";
+import { ReactComponent as FailedIcon } from "svg/failed.svg";
+import { ReactComponent as PassedIcon } from "svg/success.svg";
+import { ReactComponent as PendingIcon } from "svg/inProgress.svg";
+import { ReactComponent as SkippedIcon } from "svg/skipped.svg";
 import useStyles from "./styles";
 
 const statusBadge = (step) => {
-  const classes = useStyles();
   if (step?.status !== "completed") {
```
Comment (oumkale) [inline src/components/CustomCard/index.jsx:22]: Hi @Aman-Codes,
Will you use a switch case here?

File: src/components/Header/styles.js [hunk 3]
```diff
@@ -10,39 +10,23 @@ const useStyles = makeStyles((theme) => ({
     height: "4.9rem",
     display: "flex",
     justifyContent: "space-between",
-    background: theme.palette.header,
+    background: theme?.palette?.header,
     padding: theme.spacing(0, 7.5),
     "& *": {
-      color: theme.palette.text.secondary,
+      color: "white",
```
Comment (oumkale) [inline src/components/Header/styles.js:16]: Use here `theme` color

**Comment by oumkale** [comment]
@Aman-Codes Can you send a link so that we can have a better look?

**Comment by Aman-Codes** [comment]
> @Aman-Codes Can you send a link so that we can have a better look?

https://aman-codes.github.io/litmus-e2e/dashboard/

**Comment by oumkale** [comment]
Hi @Aman-Codes,

Make sure that all components are having proper theme in both themes.
Screenshot from 2021-11-12 18-11-07
Screenshot from 2021-11-12 18-13-39
Screenshot from 2021-11-12 18-15-09
Screenshot from 2021-11-12 18-15-13

**Comment by Aman-Codes** [comment]
Updated the PR and deployed it to https://aman-codes.github.io/litmus-e2e/dashboard/

**Comment by Aman-Codes** [comment]
The last commit adds the following enhancements:
- Removed underline from all hyperlink texts
- Added a loading spinner while fetching table data
- Updated test coverage to use radial progress chart
- Add hover effect on header text (home, nightly runs, manual runs)
- Fixed status chart background in dark theme
- Made subheading, header and dropdown outline grey in dark theme

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**